spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/async/AsyncAutoConfiguration.java

```diff
@@ -29,6 +30,7 @@
  */
 @Configuration
 @EnableConfigurationProperties(SleuthAsyncProperties.class)
+@ConditionalOnProperty(value = "spring.sleuth.async.enabled", matchIfMissing = true)
```

Comment:
Why not reuse `org.springframework.cloud.sleuth.instrument.scheduling.SleuthSchedulingProperties` and `spring.sleuth.scheduled.enabled` ?

Reply:
this feature itself is confusing really, but the properties related to what you need to stop are under the async namespace, and this feature itself, not necessarily the parent feature, is what needs to be stopped.

Reply:
for example, if you look at the class https://github.com/spring-cloud/spring-cloud-sleuth/blob/master/spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/async/SleuthAsyncProperties.java it has only one property which allows you to cherry-pick what to ignore. this is effectively saying `*`. Since this is recent, I don't know the full impact of disabling the upstream stuff. If you think it is the same, then it is curious why have the "async" property here instead of in SleuthSchedulingProperties

Reply:
I switched, though now the test is broke locally